EAD-Session-3-Assignment/src/com/metacube/servlets/ShowAllStudents.java

```diff
@@ -0,0 +1,78 @@
+package com.metacube.servlets;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet implementation class ShowAllStudents
+ */
+@WebServlet("/ShowAllStudents")
+public class ShowAllStudents extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+       
+    /**
+     * @see HttpServlet#HttpServlet()
+     */
+    public ShowAllStudents() {
+        super();
+        // TODO Auto-generated constructor stub
+    }
+
+	/**
+	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
+	 */
+	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		PrintWriter out = response.getWriter();
+		try
+		{
+			Connection connection = SQLConnection.makeConnection();
+			String query = StudentQueries.getDisplayQuery();
+			PreparedStatement statement = connection.prepareStatement(query);
+			ResultSet resultSet = statement.executeQuery();
```

Comment:
Pass the resultSet data to a jsp page and display the table data in the jsp page.

Reply:
We were told not to use jsp in these assignments